### PR TITLE
[hidapi] add new port

### PR DESCRIPTION
**Pull request by kejxu**
* consume VS `.sln` build recipe from `hidapi`
* use commit from master branch for VS `140` build recipe

related to https://github.com/microsoft/vcpkg/issues/5429

### CODE REVIEW

File: ports/hidapi/CONTROL [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Source: hidapi
+Version: 2019-08-30
```
Comment (Youw) [inline ports/hidapi/CONTROL]: I'm no expert in `vcpkg`, but this doesn't look like a library version.
Is this a common practice in `vcpkg`, to use source snapshot date as a version?

Comment (myd7349) [reply ports/hidapi/CONTROL]: https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/control-files.md#version

Comment (cbezault) [reply ports/hidapi/CONTROL]: If the library operates on a rolling release or has not made a release in a long time and there are critical updates that should be included in any vcpkg port it is acceptable to put a date as the version.

Comment (kejxu) [reply ports/hidapi/CONTROL]: @Youw I am using the current latest commit (created in 2019/08/30) in the master branch to consume `hidapi.vcxproj` (which is missing from [`0.9.0`](https://github.com/libusb/hidapi/tree/hidapi-0.9.0/windows)). As @myd7349 and @cbezault mentioned, using the date as the version number is the preferred way in this case =)

per your suggestion in the other thread, we could update this `vcpkg` port once the project is converted to use CMake build system

**Pull request merged**